Makefile

```diff
@@ -35,7 +35,7 @@ help:
 # Internal subcommands that the user should not call
 create-test-dirs:
 	mkdir -p test/packages
-	@for os in rhel6 rhel7 rhel8 centos6 centos7; do \
+	@for os in rhel6 rhel7 rhel8 centos6 centos7 centos8; do \
```

Comment:
do you think we move `rhel6 rhel7 rhel8 centos6 centos7 centos8` as a variable at the top? So that way in the future we can just edit that variable?

Reply:
Seems reasonable.  I can work on that.

Reply:
Done.